pds_doi_core/input/osti_input_util.py

```diff
@@ -0,0 +1,138 @@
+from lxml import etree
+from datetime import datetime
+
+from pds_doi_core.util.general_util import get_logger
+
+logger = get_logger('pds_doi_core.input.osti_input_util')
+
+class OSTIInputUtil:
+    def read_osti_xml(self, osti_response_text):
```

Comment:
That is good to have this code in an object. Perfect.

I would call the method just read_xml (OSTI is already in the object name)

Reply:
You are right.  No needed to duplicate information in function name.  Code has been changed.

Reply:
Ok perfect